renovate.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":dependencyDashboard",
+    ":semanticPrefixFixDepsChoreOthers",
+    "group:dotNetCore",
+    "group:monorepos",
+    "group:recommended",
+    "replacements:all",
+    "workarounds:all"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "packageRules": [
+    {
+      "groupName": "ApplicationInsights",
+      "matchPackagePatterns": [
+        "ApplicationInsights"
+      ]
+    },
+    {
+      "groupName": "Dotnet",
+      "matchPackageNames": [
+        "dotnet-sdk",
+        "mcr.microsoft.com/dotnet/aspnet",
+        "mcr.microsoft.com/dotnet/sdk"
+      ]
+    },
+    {
+      "excludePackagePrefixes": [
+        "Microsoft.Health.Fhir."
+      ],
+      "groupName": "Healthcare Shared Components",
+      "matchPackagePrefixes": [
+        "Microsoft.Health."
+      ]
+    },
+    {
+      "groupName": "Hl7",
+      "matchPackagePrefixes": [
+        "Hl7.Fhir."
+      ]
+    },
+    {
+      "groupName": "IdentityModel.Tokens",
+      "matchPackagePatterns": [
+        "IdentityModel.Tokens"
+      ]
+    },
+    {
+      "groupName": "IdentityServer4",
+      "matchPackagePrefixes": [
+        "IdentityServer4"
+      ]
+    },
+    {
+      "groupName": "Microsoft.AspNetCore.Mvc.Versioning",
+      "matchPackagePrefixes": [
+        "Microsoft.AspNetCore.Mvc.Versioning"
+      ]
+    },
+    {
```

Comment:
Should we move this up to be near the other `Microsoft.Health.` group earlier?

Reply:
Totally. I was just alphabetizing by group name lol